starboard/raspi/2/toolchain/BUILD.gn

```diff
@@ -28,26 +24,15 @@ gcc_toolchain("starboard") {
   # We use whatever 'ar' resolves to.
   ar = gcc_toolchain_ar
 
-  tail_lib_dependencies = "-l:libpthread.so.0 -l:libdl.so.2"
-
-  toolchain_args = {
-    is_clang = false
-  }
-}
-
-gcc_toolchain("target") {
-  cc = gcc_toolchain_cc
-  cxx = gcc_toolchain_cxx
-  ld = cxx
-
-  # We use whatever 'ar' resolves to.
-  ar = gcc_toolchain_ar
-
-  tail_lib_dependencies = "-l:libpthread.so.0 -l:libdl.so.2"
-
   toolchain_args = {
     is_clang = false
+    current_cpu = "arm"
+    current_os = "linux"
+    is_starboard = true
+    sysroot = "$raspi_home/busterroot"
+    use_cxx17 = true
```

Comment:
Do you not need to add a conditional using this flag? Does this alone work?

Reply:
Does https://source.corp.google.com/h/github/youtube/cobalt/+/main:build/config/compiler/BUILD.gn;l=620?q=use_cxx17&ss=h%2Fgithub%2Fyoutube%2Fcobalt%2F%2B%2Frefs%2Fheads%2Fmain get hit? Does this build without `use_cxx17=true`? Reading the code I wouldn't expect setting this flag alone to fix the issue